keel-core/src/main/kotlin/com/netflix/spinnaker/keel/telemetry/TelemetryListener.kt

```diff
@@ -115,8 +114,8 @@ class TelemetryListener(
     ).safeIncrement()
   }
 
-  @EventListener(ScheduledResourceCheckStarting::class)
```

Comment:
Is the new event in addition or instead `ScheduledResourceCheckStarting`?

Reply:
Instead of. Nothing is currently consuming `ScheduledResourceCheckStarting` events. Maybe I should just remove that class altogether.

Reply:
Addressed in 5ad23592